internal/policygen/schema.go

```diff
@@ -29,63 +29,89 @@ properties:
+      Top level configs to control what policies to generate and their enforcing scope.
     type: object
     additionalProperties: false
     properties:
-      allowed_policy_member_domains:
+      gcp_org_policies:
         description: |
-          List of domains to allow their members to be granted IAM roles.
-          gserviceaccount.com is always allowed to make sure GCP internal services work properly.
-        type: array
-        items:
-          type: string
+          Key value pairs to define the scope to apply the policies.
+        type: object
+        additionalProperties: false
+        required:
+        - parent_type
+        - parent_id
+        properties:
+          parent_type:
+            description: |
+              Type of parent GCP resource to apply the policy: can be one of "organization",
+              "folder", or "project".
+            type: string
+            pattern: ^organization|folder|project$
+
+          parent_id:
+            description: |
+              ID of parent GCP resource to apply the policy: can be one of the organization ID,
+              folder ID, or project ID according to parent_type.
+            type: string
+            pattern: ^[0-9]{8,25}$
+
+      forseti_policies:
+        description: |
+          Key value pairs to define the scope to apply the policies.
+        type: object
+        additionalProperties: false
+        required:
+        - targets
+        properties:
+          targets:
+            description: |
+              List of targets to apply the policies, e.g. organization/*,
+              organization/123/folder/456.
+            type: array
+            items:
+              type: string
 
-  gcp_organization_policies:
+  iam:
     description: |
-      Key value pairs passed to GCP Organization Policy constraint templates. If an optional
+      Key value pairs to configure IAM related policies. If an optional
       property is not specified, then the corresponding default setting will be applied.
-      For example, if ALLOWED_* is not specified, then the corresponding policy will deny all.
+      For example, if allowed_* is not specified, then the corresponding policy will deny all.
     type: object
     additionalProperties: false
     properties:
-      parent_type:
-        description: |
-          Type of parent GCP resource to apply the policy: can be one of "organization",
-          "folder", or "project".
-        type: string
-        pattern: ^organization|folder|project$
-
-      parent_id:
-        description: |
-          ID of parent GCP resource to apply the policy: can be one of the organization ID,
-          folder ID, or project ID according to parent_type.
-        type: string
-        pattern: ^[0-9]{8,25}$
-
       allowed_policy_member_domains:
         description: |
           See templates/policygen/org_policies/variables.tf.
         type: array
         items:
           type: string
 
+  network:
```

Comment:
Technically network falls under compute in GCP, but it's probably clearer to separate them. I'll leave it to your judgment

Reply:
Changed to `compute` to stick to https://github.com/GoogleCloudPlatform/magic-modules/tree/master/products